test/conformance/ingress/util.go

```diff
@@ -701,6 +701,9 @@ func CreateDialContext(t *testing.T, ing *v1alpha1.Ingress, clients *test.Client
 		if err != nil {
 			return nil, err
 		}
+		if pkgTest.Flags.IngressEndpoint != "" {
```

Comment:
Can you note that this is needed for minikube-like setups?

Reply:
@mattmoor done.